plugins/modules/web_infrastructure/jira.py

```diff
@@ -500,19 +487,19 @@ def main():
             timeout=dict(type='float', default=10),
             validate_certs=dict(default=True, type='bool'),
         ),
+        required_if=(
+            ('operation', 'create', ['project', 'issuetype', 'summary']),
+            ('operation', 'comment', ['issue', 'comment']),
+            ('operation', 'fetch', ['issue']),
+            ('operation', 'transition', ['issue', 'status']),
```

Comment:
The original code did not require `issue`.

Reply:
Yeah it did not, and wrongly so. If you look at the ``transition() `` code you'll see that it does require that for building the URI. And the only thing we can "transition" in Jira is an issue.

Should I explicitly mention this in the changelog fragment?

Reply:
Goot spot! And yes, that's definitely something that should be mentioned in the fragment.

Reply:
done